src/librustc_resolve/diagnostics.rs

```diff
@@ -843,6 +843,16 @@ Or:
 let unknown_variable = 12u32;
 let x = unknown_variable; // ok!
 ```
+
+If the item isn't available in the current module, please check you imported it:
+
+```ignore
+use foo::bar;
+bar();
+```
+
+...and check the item is public, non-private at least, to the current module.
```

Comment:
"... and check if the item is public, non-private at least, to the current module."